### ISSUE TITLE
gvim.bat (on Windows) doesn't wait on the --nofork flag (not like -f)

### ISSUE THREAD

**Issue by killerswan** [vim/vim#658]
The `C:\Windows\gvim.bat` file I have installed now is not behaving correctly on the `--nofork` flag.

That seems to be because the script is checking for only the `-f` flag (the short form):

``` batch
if NOT .%1==.-f goto noforkarg
```

And using that to call `start` with its command's `/wait` flag:

``` batch
start /w "%VIM_EXE_DIR%\gvim.exe"  %VIMARGS%
```

I'm on 1229 from [tuxproject](https://tuxproject.de/projects/vim/):

```
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Feb  1 2016 19:40:18)
MS-Windows 64-bit GUI version with OLE support
Included patches: 1-1229
```

Can anybody point me at the right file to edit to fix this in a pull req.?  :smile_cat:

**Comment by vim-ml** [comment]
On Tuesday, February 23, 2016 at 3:32:35 PM UTC-6, Kevin Cantú wrote:

> The C:\Windows\gvim.bat file I have installed now is not behaving correctly on the --nofork flag.
> 
> [Snip]
> 
> Can anybody point me at the right file to edit to fix this in a pull req.?   

src/dosinst.c

**Comment by killerswan** [comment]
Yep, that's where I put my patch: https://github.com/vim/vim/pull/659

**Comment by chrisbra** [comment]
fixed by e609ad557c15e3e5d1
